force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -416,6 +416,13 @@
         <shortDescription>Filter by {SObject Label}</shortDescription>
         <value>Filter by {0}</value>
     </labels>
+    <labels>
+        <fullName>Filter_by_Stage</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Filter by Stage</shortDescription>
+        <value>Filter by Stage</value>
+    </labels>
```

Comment:
I have reached out to Dmitry about this change.  If I remember correctly, there was concern about using the formatted version of this label due to translation issues.  Will report back.